### ISSUE TITLE
Question about your explanation of Lemma1 in your paper

### ISSUE THREAD

**Issue by haloya-0328** [PRIME-RL/Entropy-Mechanism-of-RL#23]
Thanks for your insightful work!
I have a question about your explanation of Lemma 1 in Section 3.1. In your paper, you state: 

> "That is to say, when an action a receives a high probability from the policy before updating, and its corresponding logit is also increasing after updating, then it will decrease the policy entropy."

However, this explanation seems to describe the covariance between the log-probability of the action and its logit, rather than the covariance between the log-probability and the change of logits as stated in Lemma 1.

So a more accurate explanation may be:
>" when an action a receives a high probability from the policy before updating, and receive larger positive logit updates, then it will decrease the policy entropy."

Is my understanding correct?

**Comment by Raf-Chen** [comment]
thanks for the attention.

In a word, we’re referring to the covariance between the action's log-probability and the logit update.
